src/ets/resources/config/mantis/genesis/Berlin.json

```diff
@@ -0,0 +1,15 @@
+{
+    "params" : {
+       "homesteadForkBlock" : "0x00",
+       "EIP150ForkBlock" : "0x00",
+       "EIP158ForkBlock" : "0x00",
+       "byzantiumForkBlock" : "0x00",
+       "constantinopleForkBlock" : "0x00",
+       "constantinopleFixForkBlock" : "0x00",
+       "istanbulForkBlock" : "0x00",
+       "berlinForkBlock" : "0x00",
+       "chainID" : "0x01"
+    },
+    "accounts" : {
+    }
+}
```

Comment:
just a note: mantis does not support Berlin

Reply:
I just copied these from the `besu` config. At some point we are supposed to upstream the config to https://github.com/ethereum/retesteth/tree/master/retesteth/configs/clientconfigs I think.